cwl/src/main/scala/wdl4s/cwl/Cwl.scala

```diff
@@ -13,12 +13,16 @@ sealed trait Cwl {
 }
 
 case class Workflow(
-  cwlVersion: Option[CwlVersion] = None,
-  `class`: Witness.`"Workflow"`.T = "Workflow".narrow,
+  cwlVersion: Option[CwlVersion] = Option(CwlVersion.default),
```

Comment:
Sorry, I should have been explicit that the default should be set here, not in the enumeration.

Thus this would be `cwlVersion: Option[CwlVersion] = Option(CwlVersion.Version1)` and please remove "default" from the enumeration.